irc/nickserv.go

```diff
@@ -194,12 +186,257 @@ password by supplying their username and then the desired password.`,
 			enabled:   servCmdRequiresAuthEnabled,
 			minParams: 2,
 		},
+		"get": {
+			handler: nsGetHandler,
+			help: `Syntax: $bGET <setting>$b
+
+GET queries the current values of your account settings. For more information
+on the settings and their possible values, see HELP SET.`,
+			helpShort:    `$bGET$b queries the current values of your account settings`,
+			authRequired: true,
+			enabled:      servCmdRequiresAccreg,
+			minParams:    1,
+		},
+		"saget": {
+			handler: nsGetHandler,
+			help: `Syntax: $bSAGET <account> <setting>$b
+
+SAGET queries the values of someone else's account settings. For more
+information on the settings and their possible values, see HELP SET.`,
+			helpShort: `$bSAGET$b queries the current values of another user's account settings`,
+			enabled:   servCmdRequiresAccreg,
+			minParams: 2,
+			capabs:    []string{"accreg"},
+		},
+		"set": {
+			handler: nsSetHandler,
+			help: `Syntax $bSET <setting> <value>$b
```

Comment:
We'll definitely be adding more options here. We don't wanna destroy our translations every time one's added. Can we have this string be assembled (similar to some of the others in here) on the fly with something like:

```go
settingDescriptions = []string{
// blah
}

tmpl = `Syntax $bSET <setting> <value>$b

Set modifies your account settings. The following settings ara available:

%s`

fullDesc = fmt.Sprintf(tmpl, strings.Join(settingDescriptions, "\n\n"));
```
or similar? separate each description blob so they can be translated separately